package.json

```diff
@@ -11,10 +11,13 @@
   },
   "private": true,
   "dependencies": {
+    "@bufbuild/connect-web": "^0.2.1",
     "@grpc/grpc-js": "^1.6.7",
+    "@openfeature/nodejs-sdk": "^0.3.2",
```

Comment:
Please rebase and use `@openfeature/js-sdk` which should be available as a dev dependency

Reply:
updated